build.gradle

```diff
@@ -91,7 +91,7 @@ task makeJar(type: Jar) {
     dependsOn 'generateProto'
     dependsOn compileJava
     from ('build/classes/java/main')
-    archiveName = 'compiler-fleet-proto.jar'
+    archiveName = '${rootProject.name}.jar'
```

Comment:
You'll probably want to keep `proto` suffix because this JAR really includes only protocol buffers.

Reply:
Indeed.